app/lib/apply_api/code_sets/disabilities.rb

```diff
@@ -8,10 +8,12 @@ module Disabilities
         "deaf" => ::Diversities::DEAF,
         "learning" => ::Diversities::LEARNING_DIFFICULTY,
         "long_standing" => ::Diversities::LONG_STANDING_ILLNESS,
+        "development_condition" => ::Diversities::DEVELOPMENT_CONDITION,
         "mental" => ::Diversities::MENTAL_HEALTH_CONDITION,
         "physical" => ::Diversities::PHYSICAL_DISABILITY,
         "social" => ::Diversities::SOCIAL_IMPAIRMENT,
-        "other" => ::Diversities::OTHER,
+        "no" => Diversities::NO_KNOWN_DISABILITY,
+        "opt_out" => Diversities::NOT_PROVIDED,
```

Comment:
Should this be called `not_provided`? that's what they call it and what we're *going* to have on the service.

Reply:
Is the no meant to map to not provided or opt out meant to map to not provided?

Reply:
The string "no" is what its called on the  apply api; we cant change that

Reply:
I was querying why we weren't calling it `not_provided` and instead using `opt_out`.